android/app/build.gradle

```diff
@@ -25,27 +25,33 @@ apply plugin: 'com.android.application'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
 
     lintOptions {
         disable 'InvalidPackage'
     }
 
     defaultConfig {
-        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "pl.droidsonroids.fluttrise"
+        applicationId 'pl.droidsonroids.fluttrise'
         minSdkVersion 16
-        targetSdkVersion 28
+        targetSdkVersion 29
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
+    }
+
+    signingConfigs {
+        release {
+            storeFile file("${System.getProperty('user.home')}/upload.jks")
+            keyAlias System.getenv('BITRISEIO_ANDROID_KEYSTORE_ALIAS')
+            storePassword System.getenv('BITRISEIO_ANDROID_KEYSTORE_PASSWORD')
+            keyPassword System.getenv('BITRISEIO_ANDROID_KEYSTORE_PRIVATE_KEY_PASSWORD')
+        }
     }
 
     buildTypes {
         release {
-            // TODO: Add your own signing config for the release build.
-            // Signing with the debug keys for now, so `flutter run --release` works.
-            signingConfig signingConfigs.debug
+            signingConfig signingConfigs.release
```

Comment:
czy to zadziała przy np. odpalaniu z kabla na debugu? w muwi zrobiliśmy że jak BITRISEIO_ANDROID_KEYSTORE_ALIAS nie jest ustawiony to robimy fallback do debuga